number_parser/parser.py

```diff
@@ -148,6 +153,7 @@ def parse(input_string):
 
     for token in tokens:
         compare_token = token.lower()
+
         if (compare_token.isspace() or compare_token == ""):
             if not tokens_taken:
                 current_sentence.append(token)
```

Comment:
Commenting here since not able to comment below
At [line](https://github.com/arnavkapoor/number-parser/pull/12/files?file-filters%5B%5D=.py&file-filters%5B%5D=No+extension#diff-634e39388755c9ef23f50299dc52faf5R178)
can use `elif`